src/effects/native/phasereffect.cpp

```diff
@@ -0,0 +1,180 @@
+    EffectManifest manifest;
+    manifest.setId(getId());
+    manifest.setName(QObject::tr("Phaser"));
+    manifest.setAuthor("The Mixxx Team");
+    manifest.setVersion("1.0");
+    manifest.setDescription(QObject::tr(
+                "A more complex sound effect obtained by mixing the input signal" 
+                "with a copy passed through a series of all-pass filters."));
+    
+    EffectManifestParameter* stages = manifest.addParameter();
+    stages->setId("stages");
+    stages->setName(QObject::tr("Stages"));
+    stages->setDescription("Sets number of stages.");
+    stages->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
+    stages->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
+    stages->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
+    stages->setDefault(2.0);
+    stages->setMinimum(2.0);
+    stages->setMaximum(24.0);
+
+    EffectManifestParameter* frequency = manifest.addParameter();
+    frequency->setId("frequency");
+    frequency->setName(QObject::tr("Frequency"));
+    frequency->setDescription("Controls frequency.");
+    frequency->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
+    frequency->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
+    frequency->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
+    frequency->setDefault(0.5);
+    frequency->setMinimum(0.0);
+    frequency->setMaximum(5.0);
+
+    EffectManifestParameter* depth = manifest.addParameter();
+    depth->setId("depth");
+    depth->setName(QObject::tr("Depth"));
+    depth->setDescription("Controls depth.");
+    depth->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
+    depth->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
+    depth->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
+    depth->setDefault(1.0);
+    depth->setMinimum(0.0);
+    depth->setMaximum(1.0);
+
+    EffectManifestParameter* fb = manifest.addParameter();
+    fb->setId("feedback");
+    fb->setName(QObject::tr("Feedback"));
+    fb->setDescription("Feedback");
+    fb->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
+    fb->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
+    fb->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
+    fb->setDefault(0.5);
+    fb->setMinimum(0.0);
+    fb->setMaximum(1.0);
+
+    EffectManifestParameter* sweep = manifest.addParameter();
+    sweep->setId("sweep_width");
+    sweep->setName(QObject::tr("Sweep"));
+    sweep->setDescription("Sets sweep width.");
+    sweep->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
+    sweep->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
+    sweep->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
+    sweep->setDefault(0.5);
+    sweep->setMinimum(0.0);
+    sweep->setMaximum(1.0);
+
+    EffectManifestParameter* stereo = manifest.addParameter();
+    stereo->setId("stereo");
+    stereo->setName(QObject::tr("Stereo"));
```

Comment:
Thank you. This works for me for now. Maybe we can set the shift between L and R later by a knob.